### DISCUSSION TITLE
Flutter and VisualStudio Code  Environment installation !?

### DISCUSSION THREAD

**Discussion by mandata5010** [community/community#145800] [Programming Help]
### Select Topic Area

Question

### Body

I need help installing VisualStudio Code and Flutter !?? Is there anyone here who could help, even for a small fee? This is amazing, strange, eerie ,,, help is all over the internet, but it is impossible to get it when you are in great need,,, ??? VisualStudio Code and Flutter, help, please ?!!

**Comment by Evinrrr** [comment]
For installing Visual Studio Code: u can Go to the [VS Code official website](https://code.visualstudio.com/).
For installing Flutter:
Check if your system meets the requirements on [Flutter's official documentation](https://flutter.dev/docs/get - started/install).
Download the Flutter SDK from [the official download page](https://flutter.dev/docs/development/tools/sdk/archive).
Unzip the SDK to the directory you prefer.
Set environment variables:
On Windows: Add the path of Flutter's bin directory to the PATH environment variable.
On Mac or Linux: Add the path to .bash_profile and run source ~/.bash_profile.
Run flutter doctor and follow the prompts to solve any issues.

**Reply by mecodeatlas** [reply]
Hey, @mandata5010! Did @Evinrrr's response help you resolve the issue? :)

**Reply by mandata5010** [reply]
Hi ! Sorry for the delay Evinrrr and Mecodeatlas !?
I've been having trouble with the Flutter environment. I've been struggling with it a lot!
Now it seems that Flutter/ VS is working somehow, but when I run the doctor it still shows three errors? I don't know if they are very important? Can I somehow send you a report. 
I've also installed the Flutter folder/file a few times and they seem to work (the program is a push button counter) and I've examined it?
I have Win10 and the other settings on my machine should be sufficient. I'm very interested in trying Fluttr, help me out!? 

ps1. attached is the doctor report. The program seems to work for the most part, although it shows 3 problems? Is that path correct since the doctor doesn't notice it at least? 

ps2. Could there be a dedicated area for beginners here, please??

[flutter] flutter doctor -v
[√] Flutter (Channel stable, 3.24.5, on Microsoft Windows [Version 10.0.19045.5131], locale fi-FI)
    • Flutter version 3.24.5 on channel stable at C:\src\flutter\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision dec2ee5c1f (3 weeks ago), 2024-11-13 11:13:06 -0800
    • Engine revision a18df97ca5
    • Dart version 3.5.4
    • DevTools version 2.37.3

[√] Windows Version (Installed version of Windows is version 10 or higher)

[!] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
    • Android SDK at C:\Users\markk\AppData\Local\Android\Sdk
    • Platform android-TiramisuPrivacySandbox, build-tools 33.0.1
    • ANDROID_HOME = C:\Users\markk\AppData\Local\Android\Sdk;ANDROID_HOME/tools;ANDROID_HOME/tools/bin;ANDROID_HOME/platform-tools;                
    • Java binary at: C:\Program Files\Java\jdk-11.0.16.1\bin\java
    • Java version Java(TM) SE Runtime Environment 18.9 (build 11.0.16.1+1-LTS-1)
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/to/windows-android-setup for more details.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.12.2)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.12.35521.163
    X Visual Studio is missing necessary components. Please re-run the Visual Studio installer for the "Desktop development with C++" workload, and include these components:
        MSVC v142 - VS 2019 C++ x64/x86 build tools
         - If there are multiple build tool versions available, install the latest
        C++ CMake tools for Windows
        Windows 10 SDK

[!] Android Studio (version 2022.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    X Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[√] VS Code (version 1.95.3)
    • VS Code at C:\Users\markk\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.100.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19045.5131]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 131.0.6778.87
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 131.0.2903.70

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 3 categories.

**Comment by mecodeatlas** [comment]
Hi @mandata5010,

I understand! Although the response from @Evinrrr seems valid, we realize this is something more specific and not directly related to GitHub, but rather to the installation of VSCode and Flutter. To better explain, if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Visual Studio Code FAQ](https://code.visualstudio.com/docs/supporting/FAQ). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
